lib/collective/collectors/trackjs.rb

```diff
@@ -0,0 +1,110 @@
+require 'pp'
+
+module Collective::Collectors
+  class TrackJS < Collective::Collector
+    requires :faraday
+    requires :faraday_middleware
+    requires :json
+
+    resolution '60s'
+
+    def initialize(lastSeenId = nil, options)
+      @lastSeenId = lastSeenId
+      super(options)
+    end
+
+    collect do
+      instrument_errors
+    end
+
+    private
+
+    def instrument_errors
+      count = 0
+      application = ""
+      paged("/#{customer_id}/v1/errors").each do |error|
+        application = error['application'] if count == 0
+        count += 1
+      end
+      instrument 'trackjs.url.errors', count, source: application, type: 'count'
```

Comment:
This url returns errors for all applications right? Seems like you might want to either filter by application in the url, or count the errors by application and call instrument multiple times.